README.md

```diff
@@ -13,11 +13,43 @@ This repo contains 4 jvm packages:
 
 # Quickstart
 
-All the web5 libraries are published on [JitPack](https://jitpack.io). To start simply add the following to your
-`gradle.build.kts` file:
+You can add this library you your project using Gradle or Maven. There are two ways to do so. The first is pulling the
+package from Maven Central. The second is pulling the package from JitPack.
```

Comment:
is there a reason we want to continue supporting both maven central and jitpack?

Reply:
I didn't want to replace immediately in this PR. Instead, strip it away once we are 100% sure that maven is working as expected.